packages/smooth_app/lib/pages/image_crop_page.dart

```diff
@@ -7,24 +7,27 @@ import 'package:image_picker/image_picker.dart';
 import 'package:provider/provider.dart';
 import 'package:smooth_app/themes/theme_provider.dart';
 
-Future<File?> startImageCropping(BuildContext context) async {
+Future<File?> startImageCropping(BuildContext context,
+    {File? existingImage}) async {
   final bool isDarktheme =
       Provider.of<ThemeProvider>(context, listen: false).isDarkMode(context);
   final Color? themeColor = isDarktheme
       ? Colors.black
       : Theme.of(context).appBarTheme.backgroundColor;
 
-  final ImagePicker picker = ImagePicker();
-  final AppLocalizations appLocalizations = AppLocalizations.of(context);
-  final XFile? pickedXFile = await picker.pickImage(
-    source: ImageSource.camera,
-  );
-  if (pickedXFile == null) {
-    return null;
-  }
+  late XFile? pickedXFile;
+  if (existingImage == null) {
+    final ImagePicker picker = ImagePicker();
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+    pickedXFile = await picker.pickImage(
+      source: ImageSource.camera,
+    );
+    if (pickedXFile == null) {
+      return null;
+    }
 
```

Comment:
```suggestion
}
```
Sorry I goofed while rebasing. Hopefully this is a valid fix